airbyte-config/init/src/main/java/io/airbyte/config/init/LocalDefinitionsProvider.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.init;
+
+import static io.airbyte.config.init.PatchingHelpers.addMissingCustomField;
+import static io.airbyte.config.init.PatchingHelpers.addMissingPublicField;
+import static io.airbyte.config.init.PatchingHelpers.addMissingTombstoneField;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.io.Resources;
+import io.airbyte.commons.docker.DockerUtils;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.util.MoreIterators;
+import io.airbyte.commons.yaml.Yamls;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.persistence.ConfigNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * This config persistence contains all seed definitions according to the yaml files. It is
```

Comment:
This is no longer a `config persistence`

Reply:
Done in dff32bb35c8288f09660c5368b50d22e347f6d28